profiles/common/modules/features/cce_basic_config/cce_basic_config.module

```diff
@@ -56,17 +56,19 @@ function cce_basic_config_form_system_site_information_settings_alter(&$form, &$
     '#description' => t('Choose the classification for your website. This classification respects the !url', array('!url' => l(t('IPG guidelines'), "http://ec.europa.eu/ipg/content/optimise/metadata/annex2_en.htm"))),
   );
 
+  // @codingStandardsIgnoreStart: NEPT-1304: $form_state['values'] is not populated at this stage.
```

Comment:
Is it not possible to add a tag @DrupalPracticeGeneralFormStateInputInput (or something similar :) ), to exclude that particular check instead of completely ignoring all code standard here ?

Reply:
I guess its a possibility but we need to run it through the QA team to implement it. Another option would be added one @codingStandardsIgnoreLine tag before each $form_state usage, but that seems like a lot of noise for what it seems to be some really innocuous 10 lines of code.

Reply:
Ok forget it then :)